ui/app/styles/core/forms.scss

```diff
@@ -318,6 +257,7 @@ fieldset.form-fieldset {
   border: none;
 }
 
+// ARG TODO move to helper
```

Comment:
curious about your reasoning behind moving this to a helper since it seems pretty form specific?

Reply:
Good question. The naming to me suggest helper. So something like `select-error-border`... something that follows our new helper class name standards.